client/platform/desktop/backend/ipcService.ts

```diff
@@ -29,21 +26,33 @@ export default function register() {
   });
 
   /**
-   * TODO: replace linux defaults with some kind of platform switching logic
+   * Platform-dependent methods
    */
 
+  // defaults to linux if win32 doesn't exist
+  const currentPlatform = OS.platform() === 'win32' ? win32 : linux;
+  // Initialize windows to check environment variables for %PROGRAMFILES% as
+  // it could be on another drive letter
```

Comment:
```suggestion
```

Remove or move to windows file.  Generally, document implementation details inside the function instead of at the caller.